core/block/object/objectcreator/creator.go

```diff
@@ -47,7 +47,9 @@ type Service interface {
 	CreateSmartBlockFromStateInSpace(ctx context.Context, space clientspace.Space, objectTypeKeys []domain.TypeKey, createState *state.State) (id string, newDetails *types.Struct, err error)
 
 	InstallBundledObjects(ctx context.Context, space clientspace.Space, sourceObjectIds []string, isNewSpace bool) (ids []string, objects []*types.Struct, err error)
-	app.Component
+
+	RunMigrations(space clientspace.Space)
```

Comment:
why it's here?

Reply:
Migration logic is removed from migrator